common/app/layout/slices/FixedContainers.scala

```diff
@@ -51,6 +52,7 @@ object FixedContainers {
     ("fixed/medium/fast-XII", fixedMediumFastXII),
     ("fixed/large/slow-XIV", slices(ThreeQuarterQuarter, QuarterQuarterQuarterQuarter, Ql2Ql2Ql2Ql2)),
     ("fixed/thrasher", thrasher),
+    ("fixed/showcase-single-stories", showcaseSingleStories),
```

Comment:
Does this string need to match the string in [this PR](https://github.com/guardian/facia-tool/pull/1385), `fixed-showcase`?

Reply:
Yes - that's a good point.